class_gu.py

```diff
@@ -0,0 +1,69 @@
+from enum import Enum
+
+
+class Gender(Enum):
+
+    FEMALE = "FEMALE"
+    MALE = "MALE"
+    NON_BINARY = "NON_BINARY"
+
+
+class Guest:
+
+    def __init__(self, id, name, age, city, phone_number, gender):
+        '''
+        Функція для ініціалізації основних атрибутів в класі Guest.
+        '''
+        self.id = id
+        self.name = name
+        self.age = age
+        self.city = city
+        self.phone_number = phone_number
+        self.gender = gender
+
+    def is_lucky_phone_number(self):
+        '''
+        Функція для визначення чи є номер користувача щасливим.
+        '''
+        if self.phone_number.count('7') > 3:
+            print(f"Номер користувача {self.name} є щасливим.")
+        else:
+            print(f"Номер користувача {self.name} є не щасливим.")
```

Comment:
A function must return something:

```
def is_lucky_phone_number(self):
        '''
        Функція для визначення чи є номер користувача щасливим.
        '''
        return self.phone_number.count('7') > 3
```

It will be better if it returns a boolean, it provides more flexibility

Reply:
I fixed this